prof.c

```diff
@@ -415,6 +424,75 @@ static struct {
 	} queue;
 } utracy;
 
+#if defined(UTRACY_WINDOWS)
+UTRACY_INTERNAL
+event_pipe_t create_event_pipe(bool manual_reset, int initial_state, const char* name) {
+	return CreateEvent(NULL, manual_reset, initial_state, name ? TEXT(name) : NULL);
+}
+
+UTRACY_INTERNAL
+void close_event_pipe(event_pipe_t event_pipe) {
+	CloseHandle(event_pipe);
+}
+
+UTRACY_INTERNAL
+int wait_for_event_pipe(event_pipe_t event_pipe, int timeout_ms) {
+	DWORD result = WaitForSingleObject(event_pipe, timeout_ms);
+	switch (result) {
+		case WAIT_OBJECT_0: return 0;
+		case WAIT_TIMEOUT: return 1;
+		default: return -1;
+	}
+}
+
+UTRACY_INTERNAL
+void set_event_pipe(event_pipe_t event_pipe) {
+	SetEvent(event_pipe);
+}
+#elif defined(UTRACY_LINUX)
+UTRACY_INTERNAL
+event_pipe_t create_event_pipe(bool manual_reset, int initial_state, const char* name) {
+	(void)manual_reset; // Unused on Linux
+	(void)name; // Unused on Linux
```

Comment:
What does this do?

Reply:
eventfd description:
> eventfd() creates an "eventfd object" that can be used as an event wait/notify mechanism by user-space applications

This is a "shared" implementation so that the same functions (the `event_pipe` functions) can be used on both Windows and Linux by handling the platform differences in this one place.

Basically, these are Linux implementations of what it was doing on Windows with `CreateEvent`, `SetEvent`, and `WaitForSingleObject` before.